deploy/runtime/ui/MSLF System Control.json

```diff
@@ -1354,7 +1354,7 @@
                 },
                 "X": {
                     "Type": 3.0,
-                    "Integer": 258.0
+                    "Integer": 300.0
```

Comment:
Not a big deal but different width of group boxes: 
![image](https://github.com/NREL/SAM/assets/54449384/3b554bbb-a860-4d6d-81ee-7e6a3d8fe2cc)